pnpm-lock.yaml

```diff
@@ -8686,7 +8686,7 @@ packages:
   /@types/graceful-fs@4.1.6:
     resolution: {integrity: sha512-Sig0SNORX9fdW+bQuTEovKj3uHcUL6LQKbCrrqb1X7J6/ReAbhCXRAhc+SMejhLELFj2QcyuxmUooZ4bt5ReSw==}
     dependencies:
-      '@types/node': 18.11.18
+      '@types/node': 14.18.36
```

Comment:
Assuming the move from `18.11.18` to `14.18.36` was the motivating reason for this PR?

Reply:
This one might be fixed by this PR.